### DISCUSSION TITLE
Publishing failed - 403

### DISCUSSION THREAD

**Discussion by hrozk** [codemagic-ci-cd/codemagic-docs#2718] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

-

### Steps to reproduce

I followed all the documentation on the setup. The API is also enabled for a long time, so that's not really an issue.

What can be done to fix this?

```
Publishing failed :|
Google Play failed to upload artefacts. Google Play Android Developer API has not been used in project  'number' before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/androidpublisher.googleapis.com/overview?project= 'number' then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.: {
    "error": {
        "code": 403,
        "message": "Google Play Android Developer API has not been used in project 'number' before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/androidpublisher.googleapis.com/overview?project='number' then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.",
        "status": "PERMISSION_DENIED",
        "details": [
            {
                "@type": "type.googleapis.com/google.rpc.Help",
                "links": [
                    {
                        "description": "Google developers console API activation",
                        "url": "https://console.developers.google.com/apis/api/androidpublisher.googleapis.com/overview?project='number'"
                    }
                ]
            },
            {
                "@type": "type.googleapis.com/google.rpc.ErrorInfo",
                "reason": "SERVICE_DISABLED",
                "domain": "googleapis.com",
                "metadata": {
                    "consumer": "projects/ 'number'",
                    "service": "androidpublisher.googleapis.com"
                }
            }
        ]
    }
}

```
### Expected results

I want to be able to publish to Google Play via Code Magic

### Actual results

I'm not able to publish, there's an error

**Comment by himesh-cm** [comment]
Hey @hrozk 

If the API has been enabled for an extended period. Please create a new one, as indicated in the documentation below, and try again.

https://docs.codemagic.io/yaml-publishing/google-play/#configure-google-play-api-access

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
